GHzDACs/Cleanup/dac.py

```diff
@@ -765,21 +782,33 @@ def setPolarity(self, chan, invert):
         return self.testMode(self._setPolarity, chan, invert)
 
     def setLVDS(self, cmd, sd, optimizeSD):
-        """
-        INPUTS
-        cmd - int: 2 for DAC A, 3 for DAC B
-        sd - :
-        optimizeSD - bool:
-        
-        Returns (success, MSD, MHD, t, (range(16), MSDbits, MHDbits), checkHex)
-        success - bool: 
-        MSD - int:
-        MHD - int:
-        t - :
-        range(16)
-        MSDbits - :
-        MHDbits - :
-        checkHes - :
+        """ Calibrate DAC LVDS
+        Align DAC clocks for LVDS, varying SD (sample delay).
+        :param int cmd: 2 for DAC A, 3 for DAC B
+        :param int/None sd: If int, SD value to be set. If None, SD
+                            set to value in registry. Ignored if
+                            optimizeSD=True.
+        :param bool optimizeSD: Whether to follow procedure from data
+                                sheet to determine SD
+        :return: success - If LVDS bringup successfull. MSD and MHD
+                           should only flip once with flip locations
+                           within one bit of each other. If varies,
+                           could mean clock noise.
+                 MSD - Measured sample delay. If optimizeSD, where MSD
+                       flips when MHD=SD=0. Else -1.
+                 MHD - Measured hold delay. If optimizeSD, where MHD
+                       flips when MSD=SD=0. Else -1.
+                 t - SD value set
+                 tuple - Transpose of [SD, MSD(SD), MHD(SD) for SD in range(16)]
+                 checkHex - In binary, '0bABC', where
+                            A=1: An LVDS input above the IEEE input
+                                 voltage specification limit
+                            B=1: An LVDS input below the IEEE input
+                                 voltage specification limit
+                            C=1: Sampling in correct data cycle
+        :rtype: (bool, int, int, int,
+                 tuple(list[int], list[bool], list[bool]),
+                 int)
```

Comment:
Agreed. If I were doing this again, I'd return (success, t, checkHex) and axe the MSD/MHD/cluster garbage since these don't seem to be useful. These come from when the bringup routine actually plotted where MSD and MHD flipped.

However, changing this would require changing not only the server but also the bringup routine & possibly ScanPhases code. I think this is out of scope for this PR. But certainly this should be done at some point for greater clarity in the return (and to clean up that wonderful dac_bringup return tag).